src/app/_components/_sharedcomponents/LeaderBaseBoard/LeaderBaseBoard.tsx

```diff
@@ -13,37 +13,61 @@ const LeaderBaseBoard: React.FC<ILeaderBaseBoardProps> = ({
 	const titleOpponent =
 		connectedPlayer === "ThisIsTheWay" ? "Order66" : "ThisIsTheWay";
 	//------------------------STYLES------------------------//
-
 	const containerStyle = {
 		height: "100%",
 		width: "100%",
-		justifyContent: "space-between",
+		justifyContent: "space-between"
 	};
 
 	const rowStyle = {
 		flexGrow: 1,
-		width: "100%",
+		width: "100%"
 	};
 
 	return (
 		<Grid container direction="column" sx={containerStyle}>
-			{/* Opponent's row */}
-			<Grid sx={rowStyle}>
-				<LeaderBase
-					player={getOpponent(connectedPlayer)}
-					isLobbyView={isLobbyView}
-					title={titleOpponent}
-				/>
-			</Grid>
-			{ isLobbyView ? null : <CardActionTray /> }
-			{/* Player's row */}
-			<Grid sx={rowStyle}>
-				<LeaderBase
-					player={connectedPlayer}
-					isLobbyView={isLobbyView}
-					title={connectedPlayer}
-				/>
-			</Grid>
+			{isLobbyView ? (
```

Comment:
This was the component that I was saying we should get rid of. I think having both players leader and base in a vertiacl colum like this is utlimately too restrictive. This may hold true for the LeaderBase component as well. By allowing each component direct access to the cards themselves we can be more flexible with them.